src/control/Select.tsx

```diff
@@ -20,87 +18,72 @@ export interface ISelectProps {
 @observer
 export class Select extends React.Component<ISelectProps> {
 
-    static defaultProps: Pick<ISelectProps, "label"> = {
-        label: "Choose an option"
+    static defaultProps: Pick<ISelectProps, "placeholder"> = {
+        placeholder: "Choose an option"
     };
 
     @observable private selected: string;
 
     constructor(props: ISelectProps) {
         super(props);
         let { value, children } = props;
-        if (!value) {
-            if (children) {
-                if (Array.isArray(children)) {
-                    let selectedOption = (children as Option[]).filter(option => option.props.selected);
+        if (value === void 0) {
+            const childrenCount = React.Children.count(children);
```

Comment:
React.Children.toArray or .map should already normalize the structure into an array, regardless of the number of elements.